service/src/test/java/uk/gov/hmcts/reform/fpl/utils/AgeDisplayFormatHelperTest.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.Clock;
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.AgeDisplayFormatHelper.formatAgeDisplay;
+
+/**
+ * AgeDisplayFormatHelper calculates age based on provided date and returns it in string format.
+ */
+class AgeDisplayFormatHelperTest {
+    private static final LocalDate NOW = LocalDate.now(Clock.systemDefaultZone());
```

Comment:
If the other is then using time, you could add the fixedtimeconfig here and use that instead